independent-publisher-connectors/OpenAI GPT-4/apiDefinition.swagger.json

```diff
@@ -0,0 +1,695 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenAI (V2)",
+    "description": "Connect to the OpenAI service and use the Power of GPT-4 and other models. API key must be entered as \"Bearer YOUR_API_KEY\"",

```

Comment:
@troystaylor can you please remove the word "API" from the description. you can add this information in the apiProperties in the description for the APIKey.
![image](https://user-images.githubusercontent.com/97551789/228601896-193e6877-edf9-413f-9fd2-b20d01fc43cb.png)

Reply:
Yes, I'll remove it now, but that's the default description/tooltip created by the custom connector editor.

Reply:
@troystaylor thank you so much. I meant you can add this information in either the description or the tooltip for the APIKey :-)